test/lib/omniauth/strategies/openid_connect_test.rb

```diff
@@ -343,7 +360,7 @@ def test_info
       end
 
       def test_extra
-        assert_equal({ raw_info: user_info.as_json }, strategy.extra)
+        assert_equal({ raw_info: user_info.as_json, id_token: id_token }, strategy.extra)
```

Comment:
No `.as_json` for the `id_token`?

Reply:
`user_info.as_json` has an odd name, in fact this returns a hash (https://www.rubydoc.info/gems/openid_connect/OpenIDConnect/ConnectObject#as_json-instance_method)

`id_token` already returns a hash.